src/platform/site-wide/sass/minimal.scss

```diff
@@ -23,6 +24,7 @@
 
 @import "~@department-of-veterans-affairs/formation/sass/base/b-font-faces";
 @import "~@department-of-veterans-affairs/formation/sass/base/va";
+@import "~@department-of-veterans-affairs/formation/sass/base/b-utils";
```

Comment:
This gives us [the `.va-button-link` class](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/c95a7a9ae6e554aeb2af2cab5836bd927302436d/packages/formation/sass/base/_b-utils.scss#L16-L18) which is used in the header:

https://github.com/department-of-veterans-affairs/vets-website/blob/e210cd6e2a5a6e3e7539e760692084820cd6ec75/src/platform/site-wide/header/components/OfficialGovtWebsite/index.js#L33

https://github.com/department-of-veterans-affairs/vets-website/blob/e210cd6e2a5a6e3e7539e760692084820cd6ec75/src/platform/site-wide/header/components/VeteranCrisisLine/index.js#L9